packages/site-admin/CHANGELOG.md

```diff
@@ -23,7 +23,7 @@ Initial release of the site-admin package providing a framework for building mod
 - `useLocation`: Hook for accessing current route location
 - `useMatch`: Hook for route matching functionality
 
-## Next
+## 0.1.0
```

Comment:
Do we need to re-create a new `## Next` section?

Reply:
Looking at our documentation (monorepo.md and SemVer spec), there's no explicit requirement to maintain a `## Next` section in the CHANGELOG, but anyway let's add it.